README.md

```diff
@@ -62,6 +65,66 @@ dsc_windowsfeature {'IIS':
 
 All DSC Resource names and parameters have to be in lowercase, e.g: `dsc_windowsfeature` or `dsc_name`.
 
+### Installing Packages with DSC
+
+You can install MSIs with DSC using the Puppet type `dsc_package` which maps to the `Package` DSC Resource.
+
+~~~puppet
+dsc_package{'installpython'
+  dsc_ensure    => 'Present',
+  dsc_name      => 'Python 2.7.10',
+  dsc_productid => 'E2B51919-207A-43EB-AE78-733F9C6797C2'
+  dsc_path      => 'C:\\python.msi',
+}
+~~~
+
+The `Package` DSC Resource requires the following pieces of information to install an MSI:
+
+- ProductName: The `Name` of product being installed.
+- ProductId: The `ProductCode` of the MSI, which is the unique identifier for that MSI version.
```

Comment:
nit, it sounds like you're saying `ProductCode` uniquely identifies the MSI, but during [small](https://msdn.microsoft.com/en-us/library/aa371855(v=vs.85).aspx) or [minor](https://msdn.microsoft.com/en-us/library/aa370037(v=vs.85).aspx) upgrades, you install a new MSI (with a new `PackageCode`), with the same `ProductCode` (and `UpgradeCode`).

Probably better to say something like "The ProductCode property is a unique identifier for the particular product release, represented as a string GUID, for example "{12345678-1234-1234-1234-123456789012}"" from https://msdn.microsoft.com/en-us/library/aa370854(v=vs.85).aspx